tembo-operator/src/ingress.rs

```diff
@@ -270,8 +271,24 @@ pub async fn reconcile_postgres_ing_route_tcp(
     // Check for all existing IngressRouteTCPs in this namespace
     // Filter out any that are not for this DB or do not have the correct prefix
     for ingress_route_tcp in &ingress_route_tcps {
```

Comment:
There was some conflict between ingress reconciliation in this file and in app services, so we added some logic for ignoring ingress with label `component: appService`